templates/email/credentials.email

```diff
@@ -0,0 +1,14 @@
+Hey ______ 
+
+We're excited for you to start using the new PerDiem site! We'll be rolling things out this weekend at Mind The Music - San Francisco and you'll be able to be the first to use the new site and invest in Artists at the event. 
+
+We added your information from the previous site so you will be able to login using your previous e-mail and see your investments:
+
+Login:
+Password:
+
+To login with Facebook and have a different e-mail registered, you'll need to update your e-mail on either PerDiem or Facebook. 
```

Comment:
How about:
"To login with Facebook or Google, make sure that your email on those platforms matches {{email}}."